src/Command/Process/ApplyLocal.php

```diff
@@ -73,11 +82,16 @@ public function run(InputInterface $input, OutputInterface $output)
         $this->logger->notice('Start of applying hot-fixes');
 
         $output->writeln('<info>Applying hot-fixes</info>');
+        $appliedPatches = [];
         foreach ($patches as $patch) {
             try {
                 $message = $this->applier->apply($patch->getPath(), $patch->getTitle());
                 $this->printInfo($output, $message);
+                array_push($appliedPatches, $patch);
             } catch (ApplierException $exception) {
+                $this->printInfo($output, 'Error: patch conflict happened');
```

Comment:
Use the color tags instead of status, `<error></error>`

Reply:
It also must return corrent non-zero exit code

Reply:
> Use the color tags instead of status, `<error></error>`

Will be fixed, thanks

Reply:
> It also must return corrent non-zero exit code
It throws RuntimeException which leads to non-zero exit code
https://github.com/magento-mpi/magento-cloud-patches/blob/65837e0a187dfa282e47be67c7dc1cd349643709/src/Command/Process/ApplyLocal.php#L101

Reply:
@shiftedreality fixed, please check